tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -0,0 +1,44 @@
+package e2e_test
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift/oadp-operator/tests/e2e/lib"
+)
+
+var virtTestingEnabled bool
```

Comment:
I would declare the variable and also move the os.Getenv logic to `tests/e2e/e2e_suite_test.go`